Integrations/integration-SplunkPy.yml

```diff
@@ -238,6 +256,25 @@ script:
     - contextPath: Splunk.Result
       description: The splunk search result, json array where each item is a splunk event
     description: Search Splunk for events
+  - name: splunk-submit-event
+    arguments:
+    - name: index
+      required: true
+      description: Splunk index to push data to
```

Comment:
I recommend to add to description`run splunk-print-indices command to get all indices`